addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/core/LxCategory.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.loxone.core;
```

Comment:
Yes, I actually very much prefer it that way as external libraries make it very hard for the community to participate and to deal with updates and small fixes.
So yes, I would suggest to export that package, so that it is clear that it is meant as a publicly available API.